app/Http/Controllers/Order/InvoiceController.php

```diff
@@ -370,7 +370,8 @@ public function invoiceGenerateByForm(Request $request, $user_id = '')
             $qty = $this->getQuantity($qty, $productid, $plan);
 
             $code = $request->input('code');
-            $total = $request->input('price');
+            $price = $request->input('price');
```

Comment:
if you're only using this variable to do str_replace then you can replace that variable $request->input('price')